rules/S6677/metadata.json

```
@@ -0,0 +1,19 @@
+{
+  "title": "Named placeholders should be unique",
```

Consider:
```suggestion
  "title": "Message template placeholders should be unique",
```
The uniqueness constraint also applies to numbered placeholders and adding "Message template" to the title makes it clear that we talk about the message temple syntax here.